### PR TITLE
[IMP]l10n_ar_currency_update:onchange warning when currency name is c…

### PR DESCRIPTION
**Pull request by pablohmontenegro**
…hanged.
Ticket: 51869

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/odoo-argentina-ee/pull/115).

File: l10n_ar_currency_update/models/res_currency.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from odoo import _, api, models
+
+
+class ResCurrency(models.Model):
+    _inherit = 'res.currency'
+
+    @api.onchange('name','l10n_ar_afip_code','symbol')
+    def onchange_currency_name(self):
+        return {'warning': {'title': _('Warning'), 'message': 'Cambiar el nombre de la moneda puede repercutir tanto en el servicio de la sincronización automática de tasa de cambio como en el servicio de la facturación electrónica.'}}
```
Comment (zaoral) [inline l10n_ar_currency_update/models/res_currency.py]: Not sure why we have ths "No newline in end of file" error.
Maybe there is something can be fixed in the visual code configuration.

Please could you check it?

Comment (zaoral) [inline l10n_ar_currency_update/models/res_currency.py]: Also add _() to the message
```suggestion
        return {'warning': {'title': _('Warning'), 'message': _('Cambiar el nombre de la moneda puede repercutir tanto en el servicio de la sincronización automática de tasa de cambio como en el servicio de la facturación electrónica.')}}
```

File: l10n_ar_currency_update/__manifest__.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 ##############################################################################
 {
     'name': 'Argentinian Currency Rate Update',
-    'version': "15.0.1.0.0",
+    'version': "15.0.1.1.0",
```
Comment (zaoral) [inline l10n_ar_currency_update/__manifest__.py]: In this case, we are not updating the database ( model, views or data). So we do not need to increase the module version 
```suggestion
    'version': "15.0.1.0.0",
```

**Review by zaoral — CHANGES_REQUESTED** [review]

**Review by zaoral — APPROVED** [review]

**Comment by zaoral** [comment]
@roboadhoc r+

**Pull request closed without merging**